src/include/reconcile.h

```diff
@@ -51,11 +51,17 @@ struct __wt_rec_chunk {
     WT_ITEM key;
     WT_TIME_AGGREGATE ta;
 
-    /* Saved minimum split-size boundary information. */
-    uint32_t min_entries;
-    uint64_t min_recno;
-    WT_ITEM min_key;
-    WT_TIME_AGGREGATE ta_min;
+    /* Saved split-size boundary information. */
+    uint32_t entries_before_split_boundary;
+    uint64_t recno_at_split_boundary;
+    WT_ITEM key_at_split_boundary;
+
+    /*
+     * These two time aggregates track the timestamp information before and after the split
```

Comment:
Better to add info about split boundary as part of the structure level comment.

Reply:
8c6de08dc0